kernels/volk/volk_32fc_x2_conjugate_dot_prod_32fc.h

```diff
@@ -70,6 +85,22 @@ static inline void volk_32fc_x2_conjugate_dot_prod_32fc_generic(lv_32fc_t* resul
                                                                 const lv_32fc_t* taps,
                                                                 unsigned int num_points)
 {
+    lv_32fc_t res = lv_cmake(0.f, 0.f);
+    for (unsigned int i = 0; i < num_points; ++i) {
+        res += (*input++) * lv_conj((*taps++));
+    }
+    *result = res;
```

Comment:
The `block` implementation looks a lot like we try to outsmart a compiler on optimization. And in most cases, like this one, we fail at it because compilers are just very good at this nowadays.

Reply:
So leave it in for possible speedups on older compilers? 

I converted to draft as I'm looking at an FMA implementation as well.

Reply:
That'd be great. Though, I suggest to merge this PR and open a new one for a FMA implementation.

Reply:
Agreed with @jdemel here